org.sqlitebrowser.sqlitebrowser.yml

```diff
@@ -0,0 +1,44 @@
+app-id: org.sqlitebrowser.sqlitebrowser
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: sqlitebrowser
+rename-icon: sqlitebrowser
+rename-desktop-file: sqlitebrowser.desktop
+rename-appdata-file: sqlitebrowser.desktop.appdata.xml
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=home
+  - --share=network
+cleanup:
+  - /include
+
+modules:
+  - name: sqlite3
```

Comment:
sqlite3 is in the runtime. Isn't it enough?

Reply:
libsqlite3 is required, but for some reason it also compiles without this.